Binary files Final_pkg_UEStack/Framework/build/Framework.a and Final_pkg_UEStack.barisd/Framework/build/Framework.a differ
Binary files Final_pkg_UEStack/Framework/build/log.o and Final_pkg_UEStack.barisd/Framework/build/log.o differ
diff -rNu Final_pkg_UEStack/Framework/build/makefile Final_pkg_UEStack.barisd/Framework/build/makefile
--- Final_pkg_UEStack/Framework/build/makefile	2009-11-11 03:57:04.000000000 +0100
+++ Final_pkg_UEStack.barisd/Framework/build/makefile	2012-02-06 14:19:05.000000000 +0100
@@ -14,8 +14,11 @@
 	-I/usr/local/include/glib-2.0 \
 	-I/usr/local/lib/glib-2.0/included \
 	-I/usr/local/include/gnet-2.0 \
+	-I/usr/lib/gnet-2.0/include \
+	-I/usr/include/gnet-2.0 \
 	-I/usr/local/lib/gnet-2.0/include/\
 	 -I/usr/local/include/glib-2.0 \
+	 -I/usr/lib/glib-2.0/include \
 	-I/usr/local/lib/glib-2.0/include 
 LIB = -pthread -L/usr/local/lib \
 	-lgnet-2.0 \
Binary files Final_pkg_UEStack/Framework/build/memmgr.o and Final_pkg_UEStack.barisd/Framework/build/memmgr.o differ
Binary files Final_pkg_UEStack/Framework/build/process.o and Final_pkg_UEStack.barisd/Framework/build/process.o differ
Binary files Final_pkg_UEStack/Framework/build/socket.o and Final_pkg_UEStack.barisd/Framework/build/socket.o differ
Binary files Final_pkg_UEStack/Framework/build/timer.o and Final_pkg_UEStack.barisd/Framework/build/timer.o differ
diff -rNu Final_pkg_UEStack/Framework/hdr/socket.h Final_pkg_UEStack.barisd/Framework/hdr/socket.h
--- Final_pkg_UEStack/Framework/hdr/socket.h	2009-11-11 03:57:04.000000000 +0100
+++ Final_pkg_UEStack.barisd/Framework/hdr/socket.h	2012-02-15 17:48:32.000000000 +0100
@@ -17,11 +17,24 @@
 #include <glib.h>
 #include <gnet.h>
 
-//void create_socket_and_attach(gint , GIOFunc);
+/*
+ * This file has been modified to have SOCK_STREAM capabilities
+ */
+
 GUdpSocket* create_socket_and_attach(gint , GIOFunc);
+GTcpSocket* create_tcp_socket_and_attach(gint , GIOFunc);
+
 GUdpSocket* create_socket(guint);
+GTcpSocket* create_tcp_socket(const gchar*, guint);
+
 void delete_socket(GUdpSocket *sock);
+void delete_tcp_socket(GTcpSocket *sock);
+
 void send_msg(GUdpSocket* sock_fd, gchar* buff, gint len, gint port);
+void send_stream_msg(GTcpSocket* sock_fd, gchar* buff, gint len, gint port);
+
 void recv_msg(GUdpSocket* sock_fd, gchar* buff, gint len, gint* bytes_received);
+void recv_stream_msg(GTcpSocket* sock_fd, gchar* buff, gint len, gint* bytes_received);
+
 void Die(char *msg);
 #endif
Binary files Final_pkg_UEStack/Framework/hdr/.socket.h.swp and Final_pkg_UEStack.barisd/Framework/hdr/.socket.h.swp differ
diff -rNu Final_pkg_UEStack/Framework/src/socket.c Final_pkg_UEStack.barisd/Framework/src/socket.c
--- Final_pkg_UEStack/Framework/src/socket.c	2009-11-11 03:57:04.000000000 +0100
+++ Final_pkg_UEStack.barisd/Framework/src/socket.c	2012-02-16 17:20:32.000000000 +0100
@@ -34,8 +34,51 @@
     data = (gint)socket_fd;
     g_io_add_watch_full(io_channel, 0, G_IO_IN, process_msg, &data, NULL);
 
-	return socket_fd;
+    return socket_fd;
+}
+
+/***************************************************************
+ * Function Name  :     server_accept_function 
+ * Parameters     :     server: Server socket that's been listening for incoming connections
+ *                      client: Other end of new connection
+ *                      data: User data
+ * Return         :     void.
+ * Description    :     This is called when gnet_tcp_socket_server_accept_async()
+ *                      is about to accept an incoming connection
+ ***************************************************************/
+
+void server_accept_function(GTcpSocket *server, GTcpSocket *client, gpointer process_msg)
+{
+    /* Get the IO Channel for the created Socket */
+    GIOChannel *io_channel = NULL;
+    io_channel = gnet_tcp_socket_get_io_channel(client);
+    
+    /* Subscribe to calls from IO events */
+    data = (gint) client;
+    g_io_add_watch_full(io_channel, 0, G_IO_IN, process_msg, &data, NULL);
+}
+
+/***************************************************************
+ * Function Name  :     create_tcp_socket_and_attach 
+ * Parameters     :     UE_port: port on which UE will listen.
+ * 			process_msg: call back function to be called 
+ * 			when some data is received on the socekt.
+ * Return         :     void.
+ * Description    :     Creating the socket, IO channel and attaching
+ * 			it to main loop as a source of event.
+ ***************************************************************/
+
+GTcpSocket* create_tcp_socket_and_attach(gint UE_port, GIOFunc process_msg)
+{
+    /* Create TCP socket to receive = messages from UE Manager and MME */
+    GTcpSocket *server_socket_fd = NULL;
+    server_socket_fd = gnet_tcp_socket_server_new_with_port(UE_port);
+
+    /* Asynchronously accept incoming connections and then get IO 
+     * channel in the callback that is going to be called by this function */
+    gnet_tcp_socket_server_accept_async(server_socket_fd, server_accept_function, process_msg);
 
+    return server_socket_fd;
 }
 
 /***************************************************************
@@ -54,6 +97,24 @@
 }
 
 /***************************************************************
+ * Function Name  :     create_tcp_socket 
+ * Parameters     :     port: port on which process will listen.
+ * Return         :     GUdpSocket*: Handle to the socket.
+ * Description    :     Creating the socket and bind it to the port
+ ***************************************************************/
+
+GTcpSocket* create_tcp_socket(const gchar* address, guint port)
+{
+    /* Create Inetaddr information */
+    GInetAddr* socket_inetaddr = gnet_inetaddr_new("localhost", port);
+
+    /* Create UDP socket to receive = messages from other modules*/
+    GTcpSocket *socket_fd = NULL;
+    socket_fd = gnet_tcp_socket_new(socket_inetaddr);
+    return socket_fd;
+}
+
+/***************************************************************
  * Function Name  :     delete_socket 
  * Parameters     :     sock: socket handle. 
  * Return         :     void.
@@ -67,6 +128,19 @@
 }
 
 /***************************************************************
+ * Function Name  :     delete_tcp_socket 
+ * Parameters     :     sock: socket handle. 
+ * Return         :     void.
+ * Description    :     Deleting the socket.
+ ***************************************************************/
+
+void delete_tcp_socket(GTcpSocket *sock)
+{
+    gnet_tcp_socket_delete(sock);
+    log_trace(G_LOG_LEVEL_INFO,"TCP Sock %d deleted", sock);
+}
+
+/***************************************************************
  * Function Name  :     send_msg
  * Parameters     :     socket_fd: socket handle.
  *                      buff: buffer to send.
@@ -83,6 +157,22 @@
 }
 
 /***************************************************************
+ * Function Name  :     send_stream_msg
+ * Parameters     :     socket_fd: socket handle.
+ *                      buff: buffer to send.
+ *                      len: length of the buffer.
+ *                      port: port at which data is to be sent.
+ * Return         :     void.
+ * Description    :     sending message to destination port.
+ ***************************************************************/
+
+void send_stream_msg(GTcpSocket *socket_fd, gchar* buff, gint len, gint port)
+{
+   const GInetAddr *addr = gnet_inetaddr_new("localhost.localdomain", port);
+   gnet_tcp_socket_send(socket_fd, buff,len,addr);
+}
+
+/***************************************************************
  * Function Name  :     recv_msg
  * Parameters     :     socket_fd: socket handle.
  *                      buff: msg to be received into this.
@@ -98,6 +188,22 @@
     *bytes_received = gnet_udp_socket_receive(socket_fd, buff, len, &addr);
 }
 
+/***************************************************************
+ * Function Name  :     recv_stream_msg
+ * Parameters     :     socket_fd: socket handle.
+ *                      buff: msg to be received into this.
+ *                      len: length of the received data.
+ *                      bytes_received: number of bbytes received.
+ * Return         :     void.
+ * Description    :     receiving data from socket.
+ ***************************************************************/
+
+void recv_stream_msg(GTcpSocket *socket_fd, gchar* buff, gint len, gint* bytes_received)
+{
+    GInetAddr *addr = NULL;
+    *bytes_received = gnet_tcp_socket_receive(socket_fd, buff, len, &addr);
+}
+
 //function to show error message and exit
 void Die(char *msg)
 {
Binary files Final_pkg_UEStack/Framework/src/.socket.c.swp and Final_pkg_UEStack.barisd/Framework/src/.socket.c.swp differ
diff -rNu Final_pkg_UEStack/Test/main.c Final_pkg_UEStack.barisd/Test/main.c
--- Final_pkg_UEStack/Test/main.c	1970-01-01 01:00:00.000000000 +0100
+++ Final_pkg_UEStack.barisd/Test/main.c	2012-02-15 14:46:51.000000000 +0100
@@ -0,0 +1,114 @@
+
+#include <stdio.h>
+#include "../Framework/hdr/process.h"
+#include "../Framework/hdr/commonDefines.h"
+
+extern void Start_Ue(gchar** argv);
+
+/*
+ * Prototypes local to this file
+ */
+void start_ue(unsigned long ue_id);
+void send_message(GUdpSocket* ue_socket, unsigned long ue_id, gchar* message);
+GUdpSocket* initialize_ue_manager_socket(unsigned long ue_id);
+gboolean process_ue_manager_reply(GIOChannel *source, GIOCondition condition, gpointer data);
+
+int main(int argc, char** argv)
+{
+	int i = 0;
+
+/*	if (argc != 3) {
+		printf("%s <path_to_UEProcess_executable> <UEID>\n", argv[0]);
+
+		return -1;
+	}
+*/
+	printf("Commandline arguments:\n");
+	for (i = 0; i < argc; i++) {
+		printf("%d. argument: '%s'\n", i, argv[i]);
+	}
+
+	unsigned long ue_id = atol(argv[2]);
+
+	gchar* argv_ue[2];
+	argv_ue[0] = argv[1];
+	argv_ue[1] = argv[2];
+
+	/*
+	 * Start UE
+	 */
+	//Start_Ue(argv_ue);
+
+	/*
+	 * Should we be doing this? XXX May be unnecessary
+	 */
+	// XXX start_ue(ue_id);
+
+	printf("Initializing the UE manager socket for UEID %lu\n", ue_id);
+	GUdpSocket* ue_socket = initialize_ue_manager_socket(ue_id);
+	if (ue_socket == NULL) {
+		printf("Cannot initialize UE manager socket!\n");
+		return -1;
+	}
+
+	/*
+	 * Send test commands
+	 */
+	send_message(ue_socket, ue_id, "AT+CGDCONT?");
+	send_message(ue_socket, ue_id, "AT+CGATT?");
+
+	/*
+	 * Finalize UE manager socket
+	 */
+	delete_socket(ue_socket);
+
+	return 0;
+}
+
+void start_ue(unsigned long ue_id)
+{
+	/*
+	 * Create a new `msg' for an AT command
+	 */
+	msg ue_message;
+
+	ue_message.padding = 0;
+	ue_message.msg_id = START_UE;
+	ue_message.ue_id = ue_id;
+	ue_message.Trans_id = 1; // XXX Currently this is not of use
+
+	// memcpy(
+}
+
+void send_message(GUdpSocket* ue_socket, unsigned long ue_id, gchar* message)
+{
+	/*
+	 * Create a new `msg' for an AT command
+	 */
+	msg ue_message;
+
+	ue_message.padding = 0;
+	ue_message.msg_id = AT_CMD;
+	ue_message.ue_id = ue_id;
+	ue_message.Trans_id = 1; // XXX Currently this is not of use
+
+	unsigned char message_length = strlen(message);
+	memcpy(ue_message.Data, message, message_length);
+	ue_message.Data[message_length] = 0x0D;
+
+	send_msg(ue_socket, (gchar*)&ue_message, sizeof(ue_message), UE_BASE_PORT + ue_id);
+}
+
+GUdpSocket* initialize_ue_manager_socket(unsigned long ue_id)
+{
+	// return create_socket_and_attach(UE_BASE_PORT + ue_id, process_ue_manager_reply);
+	return create_socket_and_attach(UE_MGR_PORT);
+}
+
+gboolean process_ue_manager_reply(GIOChannel *source, GIOCondition condition, gpointer data)
+{
+	printf("There is an incoming data!\n");
+
+	return TRUE;
+}
+
Binary files Final_pkg_UEStack/Test/.main.c.swp and Final_pkg_UEStack.barisd/Test/.main.c.swp differ
Binary files Final_pkg_UEStack/Test/main.o and Final_pkg_UEStack.barisd/Test/main.o differ
diff -rNu Final_pkg_UEStack/Test/Makefile Final_pkg_UEStack.barisd/Test/Makefile
--- Final_pkg_UEStack/Test/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ Final_pkg_UEStack.barisd/Test/Makefile	2012-02-08 16:30:43.000000000 +0100
@@ -0,0 +1,36 @@
+
+CC = gcc
+
+INCLUDE = -pthread -I/usr/local/include/gnet-2.0 \
+          -I/usr/local/lib/gnet-2.0/include \
+          -I/usr/local/include/glib-2.0 \
+          -I/usr/lib/glib-2.0/include \
+          -I/usr/lib/gnet-2.0/include \
+          -I/usr/include/gnet-2.0 \
+          -I/usr/local/lib/glib-2.0/included \
+          -I/usr/local/include/gnet-2.0.so \
+          -I/usr/local/lib/gnet-2.0/include \
+          -I/usr/local/include/glib-2.0 \
+          -I/usr/local/lib/glib-2.0/include \
+
+LIB = -pthread -L/usr/local/lib \
+	-lgnet-2.0 \
+	-lgthread-2.0 \
+	-lrt \
+	-lglib-2.0 
+
+CFLAGS = -g -I ../UEStack/eMM/main/hdr -I ../UEStack/eMM/msg/hdr/ \
+	 -I ../Framework/hdr/ -I ../UEStack/interfaceModule/hdr/  \
+	 -I ../UEStack/timerMgmt/hdr -I ../UEStack/eSM/main/hdr/  \
+	 -I ../UEStack/eSM/msg/hdr/ ${LIB}
+
+tester: main.o
+	${CC} ${CFLAGS} -o $@ main.o \
+	../Framework/build/Framework.a ../UEStack/eMM/msg/build/libMessaging.so
+
+main.o: main.c ../UEStack/eMM/main/hdr/emmContext.h ../Framework/build/*.o
+	${CC} ${INCLUDE} ${CFLAGS} -c main.c
+
+clean:
+	rm -f *.o tester
+
Binary files Final_pkg_UEStack/Tools/generateUSIM_context.exe and Final_pkg_UEStack.barisd/Tools/generateUSIM_context.exe differ
Binary files Final_pkg_UEStack/Tools/generateUSIM_context.o and Final_pkg_UEStack.barisd/Tools/generateUSIM_context.o differ
diff -rNu Final_pkg_UEStack/Tools/Makefile Final_pkg_UEStack.barisd/Tools/Makefile
--- Final_pkg_UEStack/Tools/Makefile	2009-11-11 03:57:04.000000000 +0100
+++ Final_pkg_UEStack.barisd/Tools/Makefile	2012-02-06 14:18:59.000000000 +0100
@@ -14,6 +14,9 @@
 INCLUDE = -pthread -I/usr/local/include/gnet-2.0 \
          -I/usr/local/lib/gnet-2.0/include/ \
         -I/usr/local/include/glib-2.0 \
+        -I/usr/lib/glib-2.0/include \
+        -I/usr/lib/gnet-2.0/include \
+	-I/usr/include/gnet-2.0 \
         -I/usr/local/lib/glib-2.0/included \
         -I/usr/local/include/gnet-2.0.so \
         -I/usr/local/lib/gnet-2.0/include/\
Binary files Final_pkg_UEStack/UeMgr/build/NvRAM_1231.txt and Final_pkg_UEStack.barisd/UeMgr/build/NvRAM_1231.txt differ
Binary files Final_pkg_UEStack/UeMgr/build/NvRAM_666.txt and Final_pkg_UEStack.barisd/UeMgr/build/NvRAM_666.txt differ
Binary files Final_pkg_UEStack/UEStack/build/BearerMgmtMsgs.o and Final_pkg_UEStack.barisd/UEStack/build/BearerMgmtMsgs.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmDeregisteredAttemptingToAttach.o and Final_pkg_UEStack.barisd/UEStack/build/emmDeregisteredAttemptingToAttach.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmDeregisteredInitiated.o and Final_pkg_UEStack.barisd/UEStack/build/emmDeregisteredInitiated.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmDeregistered.o and Final_pkg_UEStack.barisd/UEStack/build/emmDeregistered.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmDeregisteredPlmnSearch.o and Final_pkg_UEStack.barisd/UEStack/build/emmDeregisteredPlmnSearch.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmFsm.o and Final_pkg_UEStack.barisd/UEStack/build/emmFsm.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmInterface.o and Final_pkg_UEStack.barisd/UEStack/build/emmInterface.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmMain.o and Final_pkg_UEStack.barisd/UEStack/build/emmMain.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmRegisInitWaitEpsContext.o and Final_pkg_UEStack.barisd/UEStack/build/emmRegisInitWaitEpsContext.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmRegisInitWaitPdnCon.o and Final_pkg_UEStack.barisd/UEStack/build/emmRegisInitWaitPdnCon.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmRegisteredAttemptingToUpdate.o and Final_pkg_UEStack.barisd/UEStack/build/emmRegisteredAttemptingToUpdate.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmRegisteredInitiated.o and Final_pkg_UEStack.barisd/UEStack/build/emmRegisteredInitiated.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmRegistered.o and Final_pkg_UEStack.barisd/UEStack/build/emmRegistered.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmServiceRequestInitiated.o and Final_pkg_UEStack.barisd/UEStack/build/emmServiceRequestInitiated.o differ
Binary files Final_pkg_UEStack/UEStack/build/emmTAUInitiated.o and Final_pkg_UEStack.barisd/UEStack/build/emmTAUInitiated.o differ
Binary files Final_pkg_UEStack/UEStack/build/eSMFsm.o and Final_pkg_UEStack.barisd/UEStack/build/eSMFsm.o differ
Binary files Final_pkg_UEStack/UEStack/build/eSMInterface.o and Final_pkg_UEStack.barisd/UEStack/build/eSMInterface.o differ
Binary files Final_pkg_UEStack/UEStack/build/eSMMain.o and Final_pkg_UEStack.barisd/UEStack/build/eSMMain.o differ
Binary files Final_pkg_UEStack/UEStack/build/eSMTimer.o and Final_pkg_UEStack.barisd/UEStack/build/eSMTimer.o differ
Binary files Final_pkg_UEStack/UEStack/build/eSMUtils.o and Final_pkg_UEStack.barisd/UEStack/build/eSMUtils.o differ
Binary files Final_pkg_UEStack/UEStack/build/handleGMMSMsap.o and Final_pkg_UEStack.barisd/UEStack/build/handleGMMSMsap.o differ
Binary files Final_pkg_UEStack/UEStack/build/handleRABMSMsap.o and Final_pkg_UEStack.barisd/UEStack/build/handleRABMSMsap.o differ
Binary files Final_pkg_UEStack/UEStack/build/handleSMREGsap.o and Final_pkg_UEStack.barisd/UEStack/build/handleSMREGsap.o differ
diff -rNu Final_pkg_UEStack/UEStack/build/Makefile Final_pkg_UEStack.barisd/UEStack/build/Makefile
--- Final_pkg_UEStack/UEStack/build/Makefile	2009-11-11 03:57:05.000000000 +0100
+++ Final_pkg_UEStack.barisd/UEStack/build/Makefile	2012-02-06 14:18:57.000000000 +0100
@@ -20,11 +20,14 @@
 INCLUDE = -pthread -I/usr/local/include/gnet-2.0 \
          -I/usr/local/lib/gnet-2.0/include/ \
         -I/usr/local/include/glib-2.0 \
-        -I/usr/local/lib/glib-2.0/included \
+        -I/usr/local/lib/glib-2.0/include \
         -I/usr/local/include/gnet-2.0.so \
-        -I/usr/local/lib/gnet-2.0/include/\
+        -I/usr/local/lib/gnet-2.0/include/ \
          -I/usr/local/include/glib-2.0 \
         -I/usr/local/lib/glib-2.0/include \
+        -I/usr/lib/glib-2.0/include/ \
+        -I/usr/lib/gnet-2.0/include/ \
+	-I/usr/include/gnet-2.0 \
 
 LIB = -pthread -L/usr/local/lib \
         -lgnet-2.0 \
diff -rNu Final_pkg_UEStack/UEStack/build/mscUeId0.txt Final_pkg_UEStack.barisd/UEStack/build/mscUeId0.txt
--- Final_pkg_UEStack/UEStack/build/mscUeId0.txt	1970-01-01 01:00:00.000000000 +0100
+++ Final_pkg_UEStack.barisd/UEStack/build/mscUeId0.txt	2012-02-13 15:41:05.000000000 +0100
@@ -0,0 +1,2 @@
+msc { 
+ UeMgr,IFM,eSM,RABM,eMM,RRC,MME;
diff -rNu Final_pkg_UEStack/UEStack/build/mscUeId154.txt Final_pkg_UEStack.barisd/UEStack/build/mscUeId154.txt
--- Final_pkg_UEStack/UEStack/build/mscUeId154.txt	1970-01-01 01:00:00.000000000 +0100
+++ Final_pkg_UEStack.barisd/UEStack/build/mscUeId154.txt	2012-02-15 14:47:48.000000000 +0100
@@ -0,0 +1,7 @@
+msc { 
+ UeMgr,IFM,eSM,RABM,eMM,RRC,MME;
+IFM->eMM [ label = "CELL_INDICATION" ] ;
+IFM<-UeMgr [ label = "AT+CGDCONT? " ] ;
+IFM->UeMgr [ label = "AT_REPLY:ERROR" ] ;
+IFM<-UeMgr [ label = "AT+CGATT? " ] ;
+IFM->UeMgr [ label = "AT_REPLY:OK" ] ;
diff -rNu Final_pkg_UEStack/UEStack/build/mscUeId207.txt Final_pkg_UEStack.barisd/UEStack/build/mscUeId207.txt
--- Final_pkg_UEStack/UEStack/build/mscUeId207.txt	1970-01-01 01:00:00.000000000 +0100
+++ Final_pkg_UEStack.barisd/UEStack/build/mscUeId207.txt	2012-02-06 17:13:53.000000000 +0100
@@ -0,0 +1,2 @@
+msc { 
+ UeMgr,IFM,eSM,RABM,eMM,RRC,MME;
Binary files Final_pkg_UEStack/UEStack/build/NvRAM_666.txt and Final_pkg_UEStack.barisd/UEStack/build/NvRAM_666.txt differ
Binary files Final_pkg_UEStack/UEStack/build/parseESMInfoElems.o and Final_pkg_UEStack.barisd/UEStack/build/parseESMInfoElems.o differ
Binary files Final_pkg_UEStack/UEStack/build/parser.o and Final_pkg_UEStack.barisd/UEStack/build/parser.o differ
Binary files Final_pkg_UEStack/UEStack/build/PDNMsgs.o and Final_pkg_UEStack.barisd/UEStack/build/PDNMsgs.o differ
Binary files Final_pkg_UEStack/UEStack/build/RRCInterface.o and Final_pkg_UEStack.barisd/UEStack/build/RRCInterface.o differ
Binary files Final_pkg_UEStack/UEStack/build/UEProcess and Final_pkg_UEStack.barisd/UEStack/build/UEProcess differ
Binary files Final_pkg_UEStack/UEStack/build/UEProcess.o and Final_pkg_UEStack.barisd/UEStack/build/UEProcess.o differ
Binary files Final_pkg_UEStack/UEStack/build/ueTimer.o and Final_pkg_UEStack.barisd/UEStack/build/ueTimer.o differ
Binary files Final_pkg_UEStack/UEStack/build/UserInterface.o and Final_pkg_UEStack.barisd/UEStack/build/UserInterface.o differ
diff -rNu Final_pkg_UEStack/UEStack/eMM/build/makefile Final_pkg_UEStack.barisd/UEStack/eMM/build/makefile
--- Final_pkg_UEStack/UEStack/eMM/build/makefile	2009-11-11 03:57:05.000000000 +0100
+++ Final_pkg_UEStack.barisd/UEStack/eMM/build/makefile	2012-02-06 14:18:54.000000000 +0100
@@ -7,6 +7,9 @@
 INCLUDE = -pthread -I/usr/local/include/gnet-2.0 \
          -I/usr/local/lib/gnet-2.0/include/ \
         -I/usr/local/include/glib-2.0 \
+        -I/usr/lib/glib-2.0/include \
+	-I/usr/include/gnet-2.0 \
+        -I/usr/lib/gnet-2.0/include \
         -I/usr/local/lib/glib-2.0/included \
         -I/usr/local/include/gnet-2.0 \
         -I/usr/local/lib/gnet-2.0/include/\
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/AttachAcceptMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/AttachAcceptMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/AttachCompleteMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/AttachCompleteMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/AttachRejectMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/AttachRejectMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/AttachRequestMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/AttachRequestMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/CommonHeaderMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/CommonHeaderMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/GUTIReallocCommandMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/GUTIReallocCommandMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/IdentityRequestMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/IdentityRequestMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/IdentityResponseMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/IdentityResponseMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/libMessaging.so and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/libMessaging.so differ
diff -rNu Final_pkg_UEStack/UEStack/eMM/msg/build/Makefile Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/Makefile
--- Final_pkg_UEStack/UEStack/eMM/msg/build/Makefile	2009-11-11 03:57:05.000000000 +0100
+++ Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/Makefile	2012-02-06 14:18:50.000000000 +0100
@@ -25,6 +25,9 @@
         -I/usr/local/lib/gnet-2.0/include/\
          -I/usr/local/include/glib-2.0 \
         -I/usr/local/lib/glib-2.0/include \
+        -I/usr/lib/glib-2.0/include \
+        -I/usr/lib/gnet-2.0/include \
+	-I/usr/include/gnet-2.0 \
 
 LIB = -pthread -L/usr/local/lib \
         -lgnet-2.0 \
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/Message.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/Message.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/MMEDetachRequestMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/MMEDetachRequestMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/ServiceRejectMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/ServiceRejectMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/ServiceRequestMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/ServiceRequestMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/TAUAcceptMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/TAUAcceptMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/TAURejectMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/TAURejectMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/TAURequestMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/TAURequestMsg.o differ
Binary files Final_pkg_UEStack/UEStack/eMM/msg/build/UEDetachRequestMsg.o and Final_pkg_UEStack.barisd/UEStack/eMM/msg/build/UEDetachRequestMsg.o differ
diff -rNu Final_pkg_UEStack/UEStack/eSM/build/makefile Final_pkg_UEStack.barisd/UEStack/eSM/build/makefile
--- Final_pkg_UEStack/UEStack/eSM/build/makefile	2009-11-11 03:57:05.000000000 +0100
+++ Final_pkg_UEStack.barisd/UEStack/eSM/build/makefile	2012-02-06 14:18:37.000000000 +0100
@@ -18,6 +18,9 @@
 INCLUDE = -pthread -I/usr/local/include/gnet-2.0 \
 	 -I/usr/local/lib/gnet-2.0/include/ \
 	-I/usr/local/include/glib-2.0 \
+	-I/usr/lib/glib-2.0/include \
+	-I/usr/lib/gnet-2.0/include \
+	-I/usr/include/gnet-2.0 \
 	-I/usr/local/lib/glib-2.0/included \
 	-I/usr/local/include/gnet-2.0 \
 	-I/usr/local/lib/gnet-2.0/include/\
